modules/twinklearv.js

```diff
@@ -80,6 +82,21 @@ Twinkle.arv.callback = function (uid, isIP) {
 		id: 'twinkle-arv-blockwarning'
 	});
 
+	// Temporary account notice
+	if (mw.config.get('wgRelevantUserName') && /^~2/.test(mw.config.get('wgRelevantUserName'))) {
```

Comment:
can we use `mw.util.isTemporaryUser(mw.config.get('wgRelevantUserName'))` or similar here instead?